### DISCUSSION TITLE
Make the Icon editor a standalone program, that can open LabVIEW files for icon editing.

### DISCUSSION THREAD

**Discussion by JayKayAce** [ni/labview-icon-editor#205] [General]
Historically, the workflow is to open the VI/Library/Class etc and then open the Icon editor from there. I propose to add support fort the reverse process, where you can load a VI or multiple VIs into the icon editor.

Having the Icon editor support being started as a separate program capable of opening VIs and editing their icons would make it easier to focus on the VI Icons, when designing toolkits and making pallets.

The supported files should be all the standard LabVIEW files with Icons: (non-exhaustive list)

* VIs
* Controls
* Classes
* Libraries
* ...

**Comment by svelderrainruiz** [comment]
Hey @JayKayAce, i have a working prototype that decouples the icon editor from LabVIEW into a plugin on my fork.
It would be simple to create a VI that can use the same plugin. We could collaborate on this idea if you want.

**Comment by JayKayAce** [comment]
Thanks for reaching out @svelderrainruiz, when I have finished my commitment on #22 I would like to collaborate on this.

**Comment by svelderrainruiz** [comment]
Moving this to a discussion based on conversations with John Medland, Tim Robinson and Fab
